README.md

```diff
@@ -29,18 +29,13 @@ This repository contains a collection of tools and commands used for managing th
 
 ### How to install (Amazon Linux repository):
 #### Running enclaves
-  1. Install the main Nitro CLI package from the AL2 repository: `sudo
-     amazon-linux-extras install aws-nitro-enclaves-cli`.
-  2. Add yourself to the `ne` group: `sudo usermod -aG ne $USER`. You will
-     have to log out and back in for this change to take effect.
-  3. Reserve resources (memory and CPUs) for future enclaves, by editing
-     `/etc/nitro_enclaves/allocator.yaml` (or use the default configuration -
-512MB and 2 CPUs) and then starting the resource reservation service: `sudo
-systemctl start nitro-enclaves-allocator.service`.
-  4. [Recommended] If you want your resources configuration to persist across
-     reboots, enable the service: `sudo systemctl enable
-nitro-enclaves-allocator.service`.
-  5. You are now ready to go.
+  1. Ensure that your EC2 instance was created with enclave support enabled and that your system (*and container if applicable*) has read/write access to `/dev/nitro_enclaves`
```

Comment:
Could add a "." at the end of the sentence here, to be similar to the other phrases in the list.